hooks/webhook.go

```diff
@@ -101,3 +99,19 @@ func webhookURL(webhook *v1alpha1.Webhook) (string, error) {
 	}
 	return fmt.Sprintf("%s://%s.%s:%v%s", protocol, webhook.Service.Name, webhook.Service.Namespace, port, *webhook.Path), nil
 }
+
+func webhookTimeout(webhook *v1alpha1.Webhook) (time.Duration, error) {
+	zero := time.Duration(0)
```

Comment:
I think you can just compare directly against a literal `0` below. Is there some reason to prefer a named variable?

Reply:
Changed

> Is there some reason to prefer a named variable?

My complete lack of understanding of go / k8s conventions. I was using [this](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/componentconfig/v1alpha1/defaults.go#L47) as a reference.